### PR TITLE
maven release prep

### PR DESCRIPTION
**Pull request by andrewmwells-amazon**
Preparation for maven release. Make CedarJavaFFI pull cedar from crates.io. Make relevant changes to testing.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CedarJava/build.gradle [hunk 1]
```diff
@@ -86,7 +86,7 @@ dependencies {
 test {
     useJUnitPlatform()
     environment "CEDAR_INTEGRATION_TESTS_ROOT", '../cedar/cedar-integration-tests'
-    environment 'CEDAR_JAVA_FFI_LIB', '/Users/anmwells/Development/Cedar/cedar-java/CedarJavaFFI/target/debug/libcedar_java_ffi.dylib'
+//  environment 'CEDAR_JAVA_FFI_LIB', 'set to absolute path of cedar_java_ffi native library (including file extension)'
```
Comment (shaobo-he-aws) [inline CedarJava/build.gradle:89]: Nit: indentation issue here.

**Review by shaobo-he-aws — APPROVED** [review]
LGTM.

File: CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java [hunk 2]
```diff
@@ -207,6 +207,10 @@ private static class JsonEntity {
     @TestFactory
     public List<DynamicContainer> integrationTestsFromJson() throws IOException {
         List<DynamicContainer> tests = new ArrayList<>();
+        //If we can't find the `cedar` package, don't try to load integration tests.
+        if(Files.notExists(Paths.get(CEDAR_INTEGRATION_TESTS_ROOT, "corpus_tests"))) {
```
[the remark this answers is not in the document]
Comment (cdisselkoen) [reply CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java:211]: +1

**Review by cdisselkoen — CHANGES_REQUESTED** [review]
Looks good except for the issue Shaobo pointed out in comments

**Review by anwarmamat — APPROVED** [review]

File: CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java [hunk 3]
```diff
@@ -207,6 +207,10 @@ private static class JsonEntity {
     @TestFactory
     public List<DynamicContainer> integrationTestsFromJson() throws IOException {
         List<DynamicContainer> tests = new ArrayList<>();
+        //If we can't find the `cedar` package, don't try to load integration tests.
```
[the remark this answers is not in the document]
Comment (andrewmwells-amazon) [reply CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java:210]: Added `MUST_RUN_CEDAR_INTEGRATION_TESTS` env var to ensure we run tests in CI.

File: CedarJava/build.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewmwells-amazon) [reply CedarJava/build.gradle]: Decided to leave as reminder

File: CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (andrewmwells-amazon) [reply CedarJava/src/test/java/com/cedarpolicy/SharedIntegrationTests.java]: Added `MUST_RUN_CEDAR_INTEGRATION_TESTS` to ensure tests are run in CI.

File: CedarJava/src/main/java/com/cedarpolicy/AuthorizationEngine.java [hunk 4]
```diff
@@ -68,6 +68,6 @@ public interface AuthorizationEngine {
      * @return The Cedar language major version supported
      */
     static String getCedarLangVersion() {
-        return "2.0";
+        return "2.3";
```
[the remark this answers is not in the document]
Comment (andrewmwells-amazon) [reply CedarJava/src/main/java/com/cedarpolicy/AuthorizationEngine.java:71]: Leaving as-is for now.

**Review by cdisselkoen — APPROVED** [review]

**Pull request merged**